ui/src/tests/org.test.ts

```diff
@@ -0,0 +1,43 @@
+import test from "ava"
+import { Org, orgIdValidationStore } from "../org"
+import { rewiremock } from '../rewiremock'
+import { get } from 'svelte/store'
+import { ValidationStatus } from '../validation'
+
+// doesn't actually work this simply out of the box
+// rewiremock("../api").with("../__mocks__/api.ts")
```

Comment:
Didn't want to go too far down the rabbithole of third-party mocking libraries before we decide that we need one.